packages/dappmanager/src/modules/chains/drivers/ethereum2.ts

```diff
@@ -0,0 +1,145 @@
+import fetch from "node-fetch";
+import memoize from "memoizee";
+import { getPrivateNetworkAlias } from "../../../domains";
+import { urlJoin } from "../../../utils/url";
+import { InstalledPackageData } from "../../../types";
+import { ChainDataResult } from "../types";
+import { safeProgress } from "../utils";
+
+const beaconChainServiceName = "beacon-chain";
```

Comment:
You can't ensure that all clients will have the container with this name. This should be a configurable parameter via the manifest that defaults to `beacon-chain` only if not specified

Reply:
implement on https://github.com/dappnode/DNP_DAPPMANAGER/pull/963